### ISSUE TITLE
Introduction - Using OpenShift metrics

### ISSUE THREAD

**Issue by mashiro-paradeis** [openshift-labs/learn-katacoda#453]
Hi, I'm getting an error while I try to study this lesson. There's this step 'oc new-app kubernets/guestbook'. It seems the result should be a created pod, so the metrics components can start gathering information. 

But instead of a created pod, I get this error "No such file or directory"

Even trying to find something within "--docket-image="docket.io/kubernetes/guestbook:latest" there still says "No such file or directory"

I suppose there shouldn't be any problem and no need of using extra commands, so I hope you can help me this time to figure out how to deploy this app in order to understand this lesson.

![error](https://user-images.githubusercontent.com/48811979/58038510-c0b25680-7af5-11e9-8292-6e748555463c.png)

**Comment by danielhelfand** [comment]
Closing this as it was addressed by #498, but the pull request didn't automatically close due to using an `&` with the fixes keyword.

**Comment by ilkka** [comment]
The fix in #498 involves `oc patch`ing a resource, but since the `new-app` command fails, there is no `dc` resource created, right? Shouldn't the instructions instead say to create the app like

```
oc new-app --docker-image=docker.io/kubernetes/guestbook:latest --name=guestbook
```

and _then_ patch if required?
